R/init.R

```diff
@@ -82,6 +85,13 @@ init <- function(project = NULL,
   project <- renv_path_normalize(project %||% getwd())
   renv_project_lock(project = project)
 
+  # if we're not loading this project, make sure we reset
+  # the working directory and repos when we're done
+  if (!load) {
+    renv_scope_options(repos = getOption("repos"))
+    renv_scope_wd(dir = getwd())
```

Comment:
We shouldn't need to reset the wd, right? Because that's already conditioned behind the `load`?

I think we can also just fail to set the `repos` option since `repos` is already explicitly passed to `snapshot()`? Is there a bug in `init(bioconductor = TRUE)`? Since it just sets the global repos options, which isn't consulted by snapshot, that repo doesn't end up in the lockfile.

Reply:
Nice catch. Let me see if I can fix that up.